<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dicom</class>
 <widget class="QMainWindow" name="Dicom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dicom</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources/qtdcm.qrc">
    <normaloff>:/Images/dicom.xpm</normaloff>:/Images/dicom.xpm</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QtDcm" name="qtdcm" native="true"/>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QtDcmSerieInfoWidget" name="serieInfoWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QtDcmPreviewWidget" name="previewWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QtDcmImportWidget" name="importWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>779</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenDicomdir"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenDicomdir">
   <property name="icon">
    <iconset resource="../Resources/qtdcm.qrc">
     <normaloff>:/Images/folder.svg</normaloff>:/Images/folder.svg</iconset>
   </property>
   <property name="text">
    <string>Open dicomdir</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../Resources/qtdcm.qrc">
     <normaloff>:/Images/quit.svg</normaloff>:/Images/quit.svg</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="../Resources/qtdcm.qrc">
     <normaloff>:/Images/save.svg</normaloff>:/Images/save.svg</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="../Resources/qtdcm.qrc">
     <normaloff>:/Images/settings.svg</normaloff>:/Images/settings.svg</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAnglais">
   <property name="text">
    <string>Anglais</string>
   </property>
  </action>
  <action name="actionFrancais">
   <property name="text">
    <string>Fran√ßais</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtDcm</class>
   <extends>QWidget</extends>
   <header location="global">QtDcm.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtDcmSerieInfoWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtDcmSerieInfoWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtDcmPreviewWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtDcmPreviewWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtDcmImportWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtDcmImportWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resources/qtdcm.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>Dicom</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
